### ISSUE TITLE
Duplicate keys detected: 'hankey'. This may cause an update error.

### ISSUE THREAD

**Issue by pellens** [serebrov/emoji-mart-vue#152]
When hovering an icon, I get the error:

```
[Vue warn]: Duplicate keys detected: 'hankey'. This may cause an update error.
found in
---> <Category>
       <Picker>
vue.runtime.esm.js?2b0e:619
```

Known issue?

**Comment by serebrov** [comment]
I do not see this in the [demo app](https://serebrov.github.io/emoji-mart-vue/).
Do you have a code example to reproduce the issue?

**Comment by rasko-lazic** [comment]
I have encountered the same issue, so I can provide additional details.

For now, only the `:hankey:` has triggered the bug. It causes multiple renderings of the same emoji in the recent section. For some reason that emoji pops up twice, both as `:poop:` and `:hankey:`.
These are the copied values from the `emoji-mart.frequently` key in local storage:

```{
  "+1": 16,
  "grinning": 15,
  "kissing_heart": 14,
  "heart_eyes": 13,
  "laughing": 12,
  "stuck_out_tongue_winking_eye": 11,
  "sweat_smile": 10,
  "joy": 9,
  "scream": 8,
  "disappointed": 7,
  "unamused": 6,
  "weary": 5,
  "sob": 4,
  "sunglasses": 3,
  "heart": 2,
  "poop": 1,
  "hankey": 1
}

In order to replicate the bug, delete both `emoji-mart.last` and `emoji-mart.frequently` keys from local storage, and then select use `:hankey:` emoji. Refresh the page, and you should get the emoji two times in the recent section.

**Comment by serebrov** [comment]
@rasko-lazic Thanks! 

I see the issue now: the [DEFAULTS list](https://github.com/serebrov/emoji-mart-vue/blob/master/src/utils/frequently.js#L3-L20) has `poop` which is a second name for `:hankey:` emoji (also known as `:poop:`, also known as `:shit:`).

There are also no check for alternative emoji names when adding it to the list, so it ends up being listed twice.

As far as I can see, just changing `poop` to `hankey` in that `DEFAULTS` list should fix the issue.

**Comment by serebrov** [comment]
I published version `10.2.1` with a fix.
